Sources/MapboxMaps/Annotations/AnnotationOrchestrator.swift

```diff
@@ -26,7 +26,12 @@ public protocol AnnotationManager: AnyObject {
 }
 
 internal protocol AnnotationManagerInternal: AnnotationManager {
+    var delegate: AnnotationInteractionDelegate? { get }
+
     func destroy()
+
+    /// Passes an array of feature IDs to the `AnnotationInteractionDelegate.annotationManager(_:didDetectTappedAnnotations:)` delegate method.
```

Comment:
I'd either remove this line since it's for an internal method or say something like "allows distributing results of query rendered features to annotation managers"